samples/durable-task-sdks/dotnet/Agents/PromptChaining/Worker/Orchestrations/ContentCreationOrchestration.cs

```diff
@@ -0,0 +1,67 @@
+using AgentChainingSample.Activities;
+using AgentChainingSample.Shared.Models;
+using Microsoft.DurableTask;
+using Microsoft.Extensions.Logging;
+
+namespace AgentChainingSample.Orchestrations;
+
+/// <summary>
+/// Orchestrates the news article generation workflow with multiple specialized agents
+/// </summary>
+public class ContentCreationOrchestration
+{
+    private readonly ILogger<ContentCreationOrchestration> _logger;
+
+    public ContentCreationOrchestration(ILogger<ContentCreationOrchestration> logger)
```

Comment:
Orchestrations should not use dependency injection as this violates best practices. I think you got here because you're using a non-standard way of registering your orchestrations (see my other comment). The correct way to get an `ILogger` for an orchestration is to use the `context.CreateReplaySafeLogger(...)` in the `RunAsync` method.